scripts/update/mono.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 #Have I mentioned I hate mono?
 
-if [[ -f /install/.sonarr.lock ]] || [[ -f /install/.radarr.lock ]] || [[ -f /install/.lidarr.lock ]]; then
+if [[ -f /install/.sonarr.lock ]] || [[ -f /install/.radarr.lock ]]; then
```

Comment:
could remove radarr as well; no need given the radarr -> netcore update exists;  either way this is also captured in the radarr ports PR...so we'd need to deconflict then if this is merged after